### ISSUE TITLE
When I use division function, it error like this 'float' cannot be interpreted as an integer.

### ISSUE THREAD

**Issue by YZY010203** [facebookresearch/CrypTen#395]
When I use division function, it error like this 'float' cannot be interpreted as an integer.
![图片](https://user-images.githubusercontent.com/63569612/185623629-7cff7e4f-5cb6-46de-b1f2-3f5bf3915abc.png)
![图片](https://user-images.githubusercontent.com/63569612/185625047-b6af359a-8788-4bc5-9578-95f580d56c16.png)

**Comment by lvdmaaten** [comment]
Can you share a more complete repro? The following code works as expected for me (using the latest version of CrypTen, installed from source):

```python
#!/usr/bin/env python3

import crypten

crypten.init()

x_enc = crypten.cryptensor([1.0, 2.0, 3.0])
y_enc = crypten.cryptensor(2.0)
z_enc = x_enc / y_enc
print(z_enc.get_plain_text())
```
Outputs:
```
tensor([0.5000, 1.0000, 1.5000])
```

**Comment by YZY010203** [comment]
Thank you very much.  I install the module by "pip install crypten". Could you please tell me how to install from the source

**Comment by lvdmaaten** [comment]
```
git clone https://github.com/facebookresearch/CrypTen.git
cd /CrypTen/
pip install -e .
```

**Comment by YZY010203** [comment]
I have install crypten again as you say. And the situation is as follow:
![图片](https://user-images.githubusercontent.com/63569612/185794894-7953e70d-f63f-457c-a86e-6d9b8d786edb.png)

**Comment by YZY010203** [comment]
# packages in environment at /home/yzyccc/anaconda3/envs/crypten:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
absl-py                   1.2.0                    pypi_0    pypi
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
bcrypt                    3.2.2                    pypi_0    pypi
blas                      1.0                         mkl  
boto3                     1.24.43                  pypi_0    pypi
botocore                  1.27.43                  pypi_0    pypi
brotli                    1.0.9                h5eee18b_7  
brotli-bin                1.0.9                h5eee18b_7  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2022.07.19           h06a4308_0  
cachetools                5.2.0                    pypi_0    pypi
certifi                   2022.6.15       py310h06a4308_0  
cffi                      1.15.1                   pypi_0    pypi
charset-normalizer        2.1.0                    pypi_0    pypi
crypten                   0.4.0                     dev_0    <develop>
cryptography              37.0.4                   pypi_0    pypi
cycler                    0.11.0             pyhd3eb1b0_0  
dbus                      1.13.18              hb2f20db_0  
expat                     2.4.4                h295c915_0  
fontconfig                2.13.1               h6c09931_0  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.11.0               h70c0345_0  
future                    0.18.2                   pypi_0    pypi
giflib                    5.2.1                h7b6447c_0  
glib                      2.69.1               h4ff587b_1  
google-auth               2.9.1                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
grpcio                    1.48.0                   pypi_0    pypi
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
icu                       58.2                 he6710b0_3  
idna                      3.3                      pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
jmespath                  1.0.1                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h7f8727e_0  
jsonpatch                 1.32                     pypi_0    pypi
jsonpointer               2.3                      pypi_0    pypi
kiwisolver                1.4.2           py310h295c915_0  
krb5                      1.19.2               hac12032_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.38                 h1181459_1  
libbrotlicommon           1.0.9                h5eee18b_7  
libbrotlidec              1.0.9                h5eee18b_7  
libbrotlienc              1.0.9                h5eee18b_7  
libclang                  10.0.1          default_hb85057a_2  
libedit                   3.1.20210910         h7f8727e_0  
libevent                  2.1.12               h8f2d780_0  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libllvm10                 10.0.1               hbcb73fb_5  
libpng                    1.6.37               hbc83047_0  
libpq                     12.9                 h16c4e8d_3  
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.2.0                h2818925_1  
libuuid                   1.0.3                h7f8727e_2  
libwebp                   1.2.2                h55f646e_0  
libwebp-base              1.2.2                h7f8727e_0  
libxcb                    1.15                 h7f8727e_0  
libxkbcommon              1.0.1                hfa300c1_0  
libxml2                   2.9.14               h74e7548_0  
libxslt                   1.1.35               h4e12654_0  
lz4-c                     1.9.3                h295c915_1  
markdown                  3.4.1                    pypi_0    pypi
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.5.1           py310h06a4308_1  
matplotlib-base           3.5.1           py310ha18d171_1  
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0           py310h7f8727e_0  
mkl_fft                   1.3.1           py310hd6ae3a3_0  
mkl_random                1.2.2           py310h00e6091_0  
munkres                   1.1.4                      py_0  
ncurses                   6.3                  h5eee18b_3  
nspr                      4.33                 h295c915_0  
nss                       3.74                 h0370c37_0  
numpy                     1.23.1          py310h1794996_0  
numpy-base                1.23.1          py310hcba007f_0  
oauthlib                  3.2.0                    pypi_0    pypi
omegaconf                 2.2.2                    pypi_0    pypi
onnx                      1.12.0                   pypi_0    pypi
openssl                   1.1.1q               h7f8727e_0  
packaging                 21.3               pyhd3eb1b0_0  
pandas                    1.4.3                    pypi_0    pypi
paramiko                  2.11.0                   pypi_0    pypi
pcre                      8.45                 h295c915_0  
pillow                    9.2.0                    pypi_0    pypi
pip                       22.1.2          py310h06a4308_0  
ply                       3.11            py310h06a4308_0  
protobuf                  3.19.4                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pynacl                    1.5.0                    pypi_0    pypi
pyparsing                 3.0.4              pyhd3eb1b0_0  
pyqt                      5.15.7          py310h6a678d5_1  
pyqt5-sip                 12.11.0                  pypi_0    pypi
python                    3.10.4               h12debd9_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
pytz                      2022.1                   pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     23.2.0                   pypi_0    pypi
qt-main                   5.15.2               h327a75a_6  
qt-webengine              5.15.9               hd2b0992_4  
qtwebkit                  5.212                h4eab89a_4  
readline                  8.1.2                h7f8727e_1  
requests                  2.28.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
s3transfer                0.6.0                    pypi_0    pypi
scikit-learn              1.1.1                    pypi_0    pypi
scipy                     1.9.0                    pypi_0    pypi
setuptools                61.2.0          py310h06a4308_0  
sip                       6.6.2           py310h6a678d5_0  
six                       1.16.0             pyhd3eb1b0_1  
sklearn                   0.0                      pypi_0    pypi
sqlite                    3.38.5               hc218d9a_0  
tensorboard               2.9.1                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0  
toml                      0.10.2             pyhd3eb1b0_0  
torch                     1.12.0                   pypi_0    pypi
torchfile                 0.1.0                    pypi_0    pypi
torchvision               0.13.0                   pypi_0    pypi
tornado                   6.2                      pypi_0    pypi
typing-extensions         4.3.0                    pypi_0    pypi
tzdata                    2022a                hda174b7_0  
urllib3                   1.26.11                  pypi_0    pypi
visdom                    0.1.8.9                  pypi_0    pypi
websocket-client          1.3.3                    pypi_0    pypi
werkzeug                  2.2.1                    pypi_0    pypi
wheel                     0.37.1                   pypi_0    pypi
xz                        5.2.5                h7f8727e_1  
zlib                      1.2.12               h7f8727e_2  
zstd                      1.5.2                ha4553b6_0

**Comment by YZY010203** [comment]
`# packages in environment at /home/yzyccc/anaconda3/envs/crypten:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
absl-py                   1.2.0                    pypi_0    pypi
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
bcrypt                    3.2.2                    pypi_0    pypi
blas                      1.0                         mkl  
boto3                     1.24.43                  pypi_0    pypi
botocore                  1.27.43                  pypi_0    pypi
brotli                    1.0.9                h5eee18b_7  
brotli-bin                1.0.9                h5eee18b_7  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2022.07.19           h06a4308_0  
cachetools                5.2.0                    pypi_0    pypi
certifi                   2022.6.15       py310h06a4308_0  
cffi                      1.15.1                   pypi_0    pypi
charset-normalizer        2.1.0                    pypi_0    pypi
crypten                   0.4.0                     dev_0    <develop>
cryptography              37.0.4                   pypi_0    pypi
cycler                    0.11.0             pyhd3eb1b0_0  
dbus                      1.13.18              hb2f20db_0  
expat                     2.4.4                h295c915_0  
fontconfig                2.13.1               h6c09931_0  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.11.0               h70c0345_0  
future                    0.18.2                   pypi_0    pypi
giflib                    5.2.1                h7b6447c_0  
glib                      2.69.1               h4ff587b_1  
google-auth               2.9.1                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
grpcio                    1.48.0                   pypi_0    pypi
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
icu                       58.2                 he6710b0_3  
idna                      3.3                      pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
jmespath                  1.0.1                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h7f8727e_0  
jsonpatch                 1.32                     pypi_0    pypi
jsonpointer               2.3                      pypi_0    pypi
kiwisolver                1.4.2           py310h295c915_0  
krb5                      1.19.2               hac12032_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.38                 h1181459_1  
libbrotlicommon           1.0.9                h5eee18b_7  
libbrotlidec              1.0.9                h5eee18b_7  
libbrotlienc              1.0.9                h5eee18b_7  
libclang                  10.0.1          default_hb85057a_2  
libedit                   3.1.20210910         h7f8727e_0  
libevent                  2.1.12               h8f2d780_0  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libllvm10                 10.0.1               hbcb73fb_5  
libpng                    1.6.37               hbc83047_0  
libpq                     12.9                 h16c4e8d_3  
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.2.0                h2818925_1  
libuuid                   1.0.3                h7f8727e_2  
libwebp                   1.2.2                h55f646e_0  
libwebp-base              1.2.2                h7f8727e_0  
libxcb                    1.15                 h7f8727e_0  
libxkbcommon              1.0.1                hfa300c1_0  
libxml2                   2.9.14               h74e7548_0  
libxslt                   1.1.35               h4e12654_0  
lz4-c                     1.9.3                h295c915_1  
markdown                  3.4.1                    pypi_0    pypi
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.5.1           py310h06a4308_1  
matplotlib-base           3.5.1           py310ha18d171_1  
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0           py310h7f8727e_0  
mkl_fft                   1.3.1           py310hd6ae3a3_0  
mkl_random                1.2.2           py310h00e6091_0  
munkres                   1.1.4                      py_0  
ncurses                   6.3                  h5eee18b_3  
nspr                      4.33                 h295c915_0  
nss                       3.74                 h0370c37_0  
numpy                     1.23.1          py310h1794996_0  
numpy-base                1.23.1          py310hcba007f_0  
oauthlib                  3.2.0                    pypi_0    pypi
omegaconf                 2.2.2                    pypi_0    pypi
onnx                      1.12.0                   pypi_0    pypi
openssl                   1.1.1q               h7f8727e_0  
packaging                 21.3               pyhd3eb1b0_0  
pandas                    1.4.3                    pypi_0    pypi
paramiko                  2.11.0                   pypi_0    pypi
pcre                      8.45                 h295c915_0  
pillow                    9.2.0                    pypi_0    pypi
pip                       22.1.2          py310h06a4308_0  
ply                       3.11            py310h06a4308_0  
protobuf                  3.19.4                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pynacl                    1.5.0                    pypi_0    pypi
pyparsing                 3.0.4              pyhd3eb1b0_0  
pyqt                      5.15.7          py310h6a678d5_1  
pyqt5-sip                 12.11.0                  pypi_0    pypi
python                    3.10.4               h12debd9_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
pytz                      2022.1                   pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     23.2.0                   pypi_0    pypi
qt-main                   5.15.2               h327a75a_6  
qt-webengine              5.15.9               hd2b0992_4  
qtwebkit                  5.212                h4eab89a_4  
readline                  8.1.2                h7f8727e_1  
requests                  2.28.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
s3transfer                0.6.0                    pypi_0    pypi
scikit-learn              1.1.1                    pypi_0    pypi
scipy                     1.9.0                    pypi_0    pypi
setuptools                61.2.0          py310h06a4308_0  
sip                       6.6.2           py310h6a678d5_0  
six                       1.16.0             pyhd3eb1b0_1  
sklearn                   0.0                      pypi_0    pypi
sqlite                    3.38.5               hc218d9a_0  
tensorboard               2.9.1                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0  
toml                      0.10.2             pyhd3eb1b0_0  
torch                     1.12.0                   pypi_0    pypi
torchfile                 0.1.0                    pypi_0    pypi
torchvision               0.13.0                   pypi_0    pypi
tornado                   6.2                      pypi_0    pypi
typing-extensions         4.3.0                    pypi_0    pypi
tzdata                    2022a                hda174b7_0  
urllib3                   1.26.11                  pypi_0    pypi
visdom                    0.1.8.9                  pypi_0    pypi
websocket-client          1.3.3                    pypi_0    pypi
werkzeug                  2.2.1                    pypi_0    pypi
wheel                     0.37.1                   pypi_0    pypi
xz                        5.2.5                h7f8727e_1  
zlib                      1.2.12               h7f8727e_2  
zstd                      1.5.2                ha4553b6_0  
`

**Comment by YZY010203** [comment]
Thank you very much. The error has disappeared by reinstall.
